### PR TITLE
Change environment to webpack

### PR DESCRIPTION
**Pull request by ChiaraDipi**
Remove bower and grunt infrastructure and add webpack

### CODE REVIEW

File: app/src/dataHandler/namedEntitiesParser_OLD.service.js [hunk 1]
```diff
@@ -843,4 +843,4 @@ angular.module('evtviewer.dataHandler')
 	};
 
 	return parser;
-});
\ No newline at end of file
+}]);
```
Comment (giuliac89) [inline app/src/dataHandler/namedEntitiesParser_OLD.service.js:846]: Is this file necessary? Why "OLD"?

File: app/src/imageViewer/imageViewer.service.js [hunk 2]
```diff
@@ -1,46 +1,50 @@
...
+                for (var i = 0; i < lenght; i++) {
+                    var imgobj = {
+                        type: "image",
+                        url: ""
+                    };
+                    p = pages[i];
+                    pp = parsedData.getPage(p);
+                    source = pp.source;
+                    if (source !== undefined && source !== '' && source !== ' ' && source !== null) {
+                        console.log(source);
```
Comment (giuliac89) [inline app/src/imageViewer/imageViewer.service.js:42]: Remove this line

File: app/src/dataHandler/namedEntitiesParser_OLD.service.js (hunk 1 above)
Comment (ChiaraDipi) [reply app/src/dataHandler/namedEntitiesParser_OLD.service.js:846]: Don't know. I'll remove it! If someone needs to access it, there always the git history
